pkgs/tools/audio/botamusique/default.nix

```
@@ -0,0 +1,136 @@
+{ pkgs,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+  coreutils,
+  nix-prefetch-git,
+  jq,
+  nodePackages
+}:
+let
+  nodejs = pkgs.nodejs-12_x;
+  nodeEnv = import ../../../development/node-packages/node-env.nix {
+    inherit (pkgs) stdenv lib python2 runCommand writeTextFile;
+    inherit pkgs nodejs;
+    libtool = if pkgs.stdenv.isDarwin then pkgs.darwin.cctools else null;
```

```suggestion
    libtool = lib.optional pkgs.stdenv.isDarwin cctools;
```

I am not sure if this is some node2nix function but generally overwriting packages with null should be avoided.

---

Yeah these `nodeEnv` and `node-packages.nix` statements are generated by node2nix. But no `lib.optional` won't work here (for one it returns an array, and for another the argument `libtool` isn't optional).